scaffolds.json

```diff
@@ -61,5 +61,12 @@
     "commit": "5c68d818e9bccf22409feb9759021e090c055e48",
     "folder": "example",
     "type": "mobile"
+  },
+  {
+    "name": "React Native simple example",
+    "repo": "https://github.com/jribbink/fcl-react-native-scaffold",
+    "description": "React Native (Expo) example demonstrating FCL and Flow interactions.",
```

Comment:
React Native (Expo) mobile dapp example demonstrating FCL and Flow interactions.